### PR TITLE
Fixed "Error loading grades information" with empty course code

### PR DESCRIPTION
**Pull request by IsaacNguyen**
## Summary
Refer to #1043 because it's the same issue just with course code.
Error was propagating because course code was not check in this if statement.
image

## Issues

Closes #

### CODE REVIEW

File: apps/backend/.env.sample
Comment (alexanderl19) [inline apps/backend/.env.sample:1]: Why are we deleting the .env.sample? I think this is unintended.

**Review by alexanderl19 — DISMISSED** [review]
Just the one comment on the deleted file, but otherwise, LGTM

-- Kevin

File: apps/backend/.env.sample
Comment (IsaacNguyen) [reply apps/backend/.env.sample:1]: Oh yeah, my bad; I didn't realize I did that. I just added it back!

**Review by KevinWu098 — APPROVED** [review]
LGTM!

**Pull request merged**